src/main/kotlin/com/sourcegraph/cody/chat/ui/LlmDropdown.kt

```diff
@@ -59,8 +59,8 @@ class LlmDropdown(
             ?: availableModels.find { it.model == selectedFromHistory?.model }
 
     selectedItem =
-        if (selectedModel?.codyProOnly == true && isCurrentUserFree())
-            availableModels.find { it.default }
+        if (selectedModel?.isCodyProOnly() == true && isCurrentUserFree())
+            availableModels.getOrNull(0)
         else selectedModel
 
     val isEnterpriseAccount =
```

Comment:
@jamesmcnamara we need to remove this line and just set `isVisible = true` always.

Also, I noticed that there is now `enterprise` tag present, shouldn't we filter out LLMs displayed for enterprise users based on if that tag is present?

Reply:
I pushed first change to your branch already.

I'm not sure if we need the second.

Reply:
I updated this because we don't want it to be visible for enterprise users that don't have multiple models enabled. However I'm still struggling with the integration tests failing and I'm not sure how to debug them. Do you know any good tricks for figuring out why they keep timing out? I tried opening the IDE to the test file and trying the document code action but it seemed to work fine.

Also, unrelated, but do you know how to access the generated kotlin code from the Cody repo? I don't see it referenced anywhere.

Reply:
Problem with integration tests was that updated recordings were missing (requires run of the `./gradlew :recordingIntegrationTest`). I updated them.
Interestingly it seems we were not reporting that fact properly, I will make sure we fixed that.

> Also, unrelated, but do you know how to access the generated kotlin code from the Cody repo? I don't see it referenced anywhere.

Right now they are just working as a manual reference, BUT it will change very soon as there is @RXminuS [PR in flight ](https://github.com/sourcegraph/jetbrains/pull/1898)which adress that.